### DISCUSSION TITLE
information about baram specific openfoam implementations

### DISCUSSION THREAD

**Discussion by flowgeek** [nextfoam/baram#41] [General]
hi,

are there somewhere informations about the baram specifc modifications of openfoam?

in the system files i find some new options like:
"BJLimited"
"momentumReconGrad"

and there are some "special" solvers 

cheers
alex

**Comment by jiban** [comment]
Hello, Alex.
Unfortunately, additional information is not available for now.
By the way, we are preparing a public repository for _OpenFOAM-N_, a modification of OpenFOAM, and it will be available in the near future.
Yet, if you have a specific question, please don't hesitate let me know.
For example, _BJLimited_ is for _Barth-Jespersen_’s limiter.
Thank you.
